voice/methods/calls/deleteCallsCallIdRecordingsRecordingId.md

```diff
@@ -0,0 +1,37 @@
+{% method %}
+
+## Delete Recording
+Delete both the metadata and the media of a specific recording.
```

Comment:
~specific~

Reply:
I understand wanting to be brief in the navigation table's description, but I like being explicit here actually.
I do think I prefer this though:
```suggestion
Delete both the metadata and the media of the specified recording.
```

Reply:
That change is better. I still think it's unnecessary though. Nobody is going to assume you mean that this endpoint deletes some random recording from your account

Reply:
I don't think they would, but I think that technical writing is the place to be precise and perhaps even unnecessarily unambiguous.